packages/docs/blog/2023-08-08-system-strings.md

```diff
@@ -0,0 +1,206 @@
+---
+slug: demystifying-fhir-system-strings
+title: Demystifying FHIR System Strings
+authors:
+  name: Rahul Agarwal
+  title: Medplum Core Team
+  url: https://github.com/rahul1
+  image_url: https://github.com/rahul1.png
+  tags: [fhir-datastore]
+---
+
+import CodeBlock from '@theme/CodeBlock'
+
+# Demystifying FHIR System Strings
+
+One of the main sources of confusion when starting out with FHIR are `system` strings.
+
+This field is ubiquitous across FHIR elements, but many developers who are new to healthcare don't understand its purpose or how to set it properly.
```

Comment:
This field is ubiquitous across FHIR elements, but many developers who are new to healthcare don't understand its purpose or how to set it properly.  (Suggest: adding sentence: Even the [sample data](/docs/tutorials/importing-sample-data) we provide for initial prototyping has many system identifiers, which means they are often needed even for a basic implementation.